### ISSUE TITLE
Quiz (And other buttons) Should be buttons

### ISSUE THREAD

**Issue by cycomachead** [beautyjoy/llab#21]
That is their tags should be `<button>` rather than `<div>` which is somewhat better for accessibility as well as code readability.
